scipy/special/specfun/specfun.f

```diff
@@ -1426,7 +1426,12 @@ SUBROUTINE CVA2(KD,M,Q,A)
         IMPLICIT DOUBLE PRECISION (A-H,O-Z)
         IF (M.LE.12.OR.Q.LE.3.0*M.OR.Q.GT.M*M) THEN
             CALL CV0(KD,M,Q,A)
-            IF (Q.NE.0.0D0.AND.M.NE.2) CALL REFINE(KD,M,Q,A)
+            IF (Q.NE.0.0D0.AND.M.NE.2) THEN
+               CALL BISECTION_REFINE(KD,M,Q,A)
+               A_COPY=A
+               CALL REFINE(KD,M,Q,A)
+               IF (ABS(1.0 - A/A_COPY).GT.1.0D-1) A=A_COPY
```

Comment:
Might be a bit ad-hoc but `REFINE` uses secant method which sometimes tends to move towards a far off root resulting in bumps in graphs as can be seen in https://github.com/scipy/scipy/issues/14526
Therefore I ignore the values generated by `REFINE` if the relative difference is too large. `BISECTION_REFINE` produces good enough values anyways.